kano_settings/system/display.py

```diff
@@ -518,3 +587,182 @@ def set_gfx_driver(enabled):
                 logger.error("Error restoring fpturbo_config", exception=e)
     end_config_transaction()
     set_setting('Use_GLX', enabled)
+
+
+def override_models(edid, model):
```

Comment:
Do we still need any of these functions below?

Reply:
Yup :)